storm-core/src/ui/public/flux.html

```diff
@@ -0,0 +1,139 @@
+<html>
```

Comment:
I think @vesense means that Storm UI index page should have a link to go to flux page. You're adding reverse link, which is already missing.

And I think we might want to port back to 1.x but not necessary to 1.0.x. Let's keep the bugfix version line.